x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_overview/analytics_overview.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { EuiButtonTo } from '../../../shared/react_router_helpers';
+import { COLLECTION_CREATION_PATH } from '../../routes';
+
+import { EnterpriseSearchAnalyticsPageTemplate } from '../layout';
+
+import { AnalyticsCollectionTable } from './analytics_collection_table';
+import { AnalyticsCollectionsLogic } from './analytics_collections_logic';
+
+export const baseBreadcrumbs = [
+  i18n.translate('xpack.enterpriseSearch.analytics.collections.breadcrumb', {
+    defaultMessage: 'Analytics Collections',
+  }),
+];
+
+export const AnalyticsOverview: React.FC = () => {
+  const { fetchAnalyticsCollections, setIsFirstRequest } = useActions(AnalyticsCollectionsLogic);
+  const { analyticsCollections, isLoading, hasNoAnalyticsCollections } =
+    useValues(AnalyticsCollectionsLogic);
+
+  useEffect(() => {
+    setIsFirstRequest();
```

Comment:
question: I'm assuming this was just copied, but is whether this is the first request actually used anywhere?

Reply:
unless there is a need to show an empty page (since there is no collection) or first load, we probably don't need it. 

Also I think there is already a `hasNoAnalyticsCollections` field from the backend call that can be used.
edit: already used at line 80

Reply:
Yeh I fumbled this. I should of removed it because right now it's not needed. Its enough for us to rely on `isLoading` and empty array for analytics collection to display the empty state.